src/pages/DummyEvents.jsx

```diff
@@ -174,9 +192,31 @@ const DummyEvents = () => {
     );
   };
 
+  useEffect(() => {
+    if (!fuse) {
+      return;
+    }
+    console.log(name);
+    let ands = [];
+    if (name) ands.push({ name: name });
+    if (location) ands.push({ location: location });
+    if (date) ands.push({ date: date });
+
+    let result;
+    if (ands.length > 0) {
+      const fuseResult = fuse.search({ $and: ands });
+      console.log(fuseResult);
+      // If we want to filter by score:
+      // result = fuseResult.filter(item => item.score <= 0.5).map(item => item.item);
+      result = fuseResult.map(item => item.item);
+    } else result = events;
+    console.log(result);
+    setDisplayEvents(result);
+  }, [name, location, date, fuse]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'events'. Either include it or remove the dependency array.